test/models/purpose_status_test.rb

```diff
@@ -2,12 +2,16 @@
 
 module Katello
   module PurposeStatusTests
-    def test_status_matched
-      purpose_mock = mock(purpose_method => :matched)
+    def setup
+      host.organization.stubs(:simple_content_access?).returns(false)
```

Comment:
I think this stub + setup method is no longer needed!

Reply:
Cleaned up the method since we changed things around in the other areas.